exonum/src/blockchain/config.rs

```diff
@@ -181,6 +189,13 @@ impl StoredConfiguration {
     /// JSON. Additionally, this method performs a logic validation of the
     /// configuration. The method returns either the result of execution or an error.
     pub fn try_deserialize(serialized: &[u8]) -> Result<Self, JsonError> {
+        use crypto::SIGNATURE_LENGTH;
+        use messages::HEADER_LENGTH;
```

Comment:
AFAIK we rather prefer to keep all the imports at the beginning of the file. @DarkEld3r am I right?

Reply:
I've reviewed uses of `use` and it seems so. There are some occasional imports inside functions, but most of them are in tests or are localized wildcard imports of enum variants. I'll move these ones to the beginning of the file.